### DISCUSSION TITLE
Simulcast

### DISCUSSION THREAD

**Discussion by Type1J** [ralfbiedert/openh264-rs#54] [General]
Hello, I am using this library to send a video stream in a multiple client application already, but in certain network conditions I have found that I need to reduce the resolution in order to make sure that the network can keep up.

Unfortunately, this means that all clients must suffer the loss in resolution if a client from the slow network connects.

I have heard of something called "simulcast" that OpenH264 supports for encoding more than 1 output stream for a single input source. How do I use this binding library to get multiple output streams at different bitrates?

**Comment by ralfbiedert** [comment]
> I have heard of something called "simulcast" that OpenH264 supports for encoding more than 1 output stream for a single input source. How do I use this binding library to get multiple output streams at different bitrates?

Unfortunately I have never touched simulcast, I also don't know how to produce more than one output stream. 

That said, if it exists there's probably a (raw) initialization option. I'd also expect your bitstream result to have more layers / channels than usual, and you probably have to demux them somehow and handle their distribution. 

Please let us know what you find out.
